templates/inc/footer.html

```diff
@@ -23,6 +23,8 @@
                 <div class="global-footer-row">
                         <p class="terms-para"><a class="terms-kit"
                               href="/contribute.html">Contributing to Kotlin</a></p>
+                    <p class="terms-para"><a class="terms-kit"
```

Comment:
Do we need both links “ Releases” and “EAP” in footer?
May be EAP can accessed from new release page?

Reply:
I'd leave them both at least for some time, to avoid confusing people who know about the EAP page.

Reply:
I think it's not so confusing, if when eap replaced by releases in same place in footer.
Much confuse if link just will be dropped? no?
Actually We just doesn't have much space for link in new WH footer.

Reply:
Actually, I planned to replace the _EAP_ link with _Releases_ in our new footer. We discussed this with Denis Voronov. I agree that there should be one link in the footer.
I have also added one more link to EAP under _Release details_.

Reply:
👍